package.json

```diff
@@ -20,7 +21,7 @@
     "start": "storybook dev --no-version-updates -p 6006",
     "tag": "utils/scripts/tag.mjs",
     "test": "jest --config=utils/test/jest.config.js",
-    "test:ci": "npm run lint && npm run format:all && npm exec tsc && npm test -- --coverage",
+    "test:ci": "npm test -- --coverage --runInBand --silent",
```

Comment:
`--silent`: muting console warnings to avoid [massive and unhelpful logs](https://github.com/zendeskgarden/react-components/actions/runs/9462583243/job/26066206181#step:6:127).

I'm still researching the best options to speed up Jest on GH runners.